lib/build_manager.py

```diff
@@ -43,9 +43,9 @@ def _build_packages(self, distro, packages):
         """
```

Comment:
I was not able to find a list of unique IDs defined by the Linux Standard Base, so I don't think that's the case. That's why I decided to drop this.